src/master/develop/transactions.md

```diff
@@ -1,6 +1,89 @@
 # Transactions
+Immudb supports transactions both on key-value and SQL level, but interactive transactions are supported only on SQL with the exception of `execAll` method, that provides some additional properties.
 
-`GetAll`, `SetAll` and `ExecAll` are the foundation of transactions in immudb. They allow the execution of a group of commands in a single step, with two important guarantees:
+## SQL interactive transactions
+Interactive transactions are a way to execute multiple SQL statements in a single transaction having the possibility to mix application logic with SQL statements.
```

Comment:
transaction. This makes possible to delegate application logic to SQL statements - a very common use case is for example checking if the balance > 0 before making a purchase.